pkg/components/components.go

```diff
@@ -39,9 +39,20 @@ type StrategyDetailsDeployment struct {
 	ClusterPermissions []StrategyDeploymentPermissions `json:"clusterPermissions,omitempty"`
 }
 
-const hcoName = "hyperconverged-cluster-operator"
+const (
+	hcoName             = "hyperconverged-cluster-operator"
+	kubevirtVersionEnvV = "KUBEVIRT_VERSION"
+	cdiVersionEnvV      = "CDI_VERSION"
+	cnaoVersionEnvV     = "NETWORK_ADDONS_VERSION"
+	sspVersionEnvV      = "SSP_VERSION"
+	nmoVersionEnvV      = "NMO_VERSION"
+	hppoVersionEnvV     = "HPPO_VERSION"
+)
 
-func GetDeployment(namespace, image, imagePullPolicy, conversionContainer, vmwareContainerString, smbios, machinetype string) appsv1.Deployment {
+func GetDeployment(
```

Comment:
This function takes a lot of parameters. I found it confusing. Can we pass a name-value pairs array instead? we can build it in different method and pass the result.
This approach will also is adding or dropping components.